frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.module.css

```diff
@@ -1,5 +1,6 @@
 .tabHeaderContent {
   display: flex;
+  align-items: flex-start;
```

Comment:
Maybe this could be solved with css on the helpText element instead of the parent? That way we leave less room for a regression because of a misunderstanding. For example something like `height: fit-content`?

Reply:
Will fix!